CesiumUtility/test/TestUri.cpp

```diff
@@ -96,3 +96,65 @@ TEST_CASE("Uri::resolve") {
           false,
           false) == "http://www.example.com/page/test");
 }
+
+TEST_CASE("Uri::escape") {
+  CHECK(Uri::escape("foo") == "foo");
+  CHECK(Uri::escape("foo/bar") == "foo%2Fbar");
+  CHECK(Uri::escape("🤞") == "%F0%9F%A4%9E");
```

Comment:
I had no idea our code could show emojis, very cool! (or is it?)

Reply:
Isn't UTF-8 great?